packages/python/package.json

```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "build": "node ../../utils/build-builder.mjs",
-    "test": "jest --reporters=default --reporters=jest-junit --env node --verbose --runInBand --bail",
+    "test": "VERCEL_FORCE_STREAMING_RUNTIME=1 jest --reporters=default --reporters=jest-junit --env node --verbose --runInBand --bail",
```

Comment:
This is failing on Windows (can't set env vars at the beginning of a command like that).